scene/gui/popup_menu.cpp

```diff
@@ -1020,6 +1100,14 @@ void PopupMenu::_notification(int p_what) {
 			if (system_menu_id != NativeMenu::INVALID_MENU_ID) {
 				bind_global_menu();
 			}
+
+			if (!Engine::get_singleton()->is_editor_hint() && !GLOBAL_GET("display/window/per_pixel_transparency/allowed")) {
+				Ref<StyleBoxFlat> sb = theme_cache.panel_style;
+				if ((!sb.is_null() && sb->get_shadow_size() > 0) || sb->get_corner_radius(CORNER_TOP_LEFT) > 0 || sb->get_corner_radius(CORNER_TOP_RIGHT) > 0 ||
```

Comment:
```suggestion
				if ((sb.is_valid() && sb->get_shadow_size() > 0) || sb->get_corner_radius(CORNER_TOP_LEFT) > 0 || sb->get_corner_radius(CORNER_TOP_RIGHT) > 0 ||
```
This isn't safe, the rest of it can be called on an invalid ref, so there needs to be more checks

Reply:
Are the changes good enough?